lasagne/layers/recurrent.py

```diff
@@ -1304,15 +1361,20 @@ def get_output_for(self, inputs, **kwargs):
         ----------
         inputs : list of theano.TensorType
             `inputs[0]` should always be the symbolic input variable.  When
-            this layer has a mask input (i.e. was instantiated with
-            `mask_input != None`, indicating that the lengths of sequences in
-            each batch vary), `inputs` should have length 2, where `inputs[1]`
-            is the `mask`.  The `mask` should be supplied as a Theano variable
+            this layer has a mask input (i.e. was instantiated with `mask_input
+            != None`, indicating that the lengths of sequences in each batch
+            vary), `inputs` should have length at least 2, where `inputs[1]` is
+            the `mask`.  The `mask` should be supplied as a Theano variable
```

Comment:
What I changed here was to specify "length at least 2" rather than "length 2". I have changed it back for now. Though I think it is a necessary addition for clarity. Thoughts?